cypress/integration/dashboard/events.js

```diff
@@ -5,8 +5,6 @@ const testEvent = {
   streamingUrl: 'https://test.event.org/video',
   capacity: '10',
   tags: 'Test, Event, Tag',
-  startAt: '2022-01-01T00:01',
-  endAt: '2022-01-02T00:02',
```

Comment:
Github won't let me suggest, here, but I'd like to keep these so that we confirm that it's possible to enter in a date just by typing.

I tested locally and 
```
  startAt: 'January 1, 2022 12:01 AM',
  endAt: 'January 2, 2022 12:02 AM',
  ```
  worked nicely.  I'll add a comment below about how the tests would need to change.